Sources/TuistKit/Graph/GraphNode.swift

```diff
@@ -131,6 +134,49 @@ enum PrecompiledNodeError: FatalError, Equatable {
     }
 }
 
+class SDKNode: GraphNode {
+    enum Error: Swift.Error {
+        case invalidExtension(String?)
+    }
+
+    enum `Type`: String {
+        case framework, library
+
+        init(from string: String) throws {
+            guard let type = Type(rawValue: string)
+            else { throw Error.invalidExtension(string) }
+
+            self = type
+        }
+    }
+
+    let name: String
+    let status: SDKStatus
+    let type: Type
+
+    init(name: String, status: SDKStatus) throws {
+        let sdk = AbsolutePath("/\(name)")
+
+        guard let string = sdk.extension
+        else { throw Error.invalidExtension(sdk.extension) }
+
+        self.name = name
+        self.status = status
+        type = try Type(from: string)
+
+        let path: AbsolutePath
+
+        switch type {
+        case .framework:
+            path = AbsolutePath("/System/Library/Frameworks").appending(component: name)
+        case .library:
```

Comment:
by library do you mean the `.tbd` libraries? 


Reference: [XcodeGen](https://github.com/yonaskolb/XcodeGen/blob/d4797c3ff63dac780c03e92a5c6eed4407199f65/Sources/XcodeGenKit/PBXProjGenerator.swift#L545)

Reply:
Yer, I've not done anything with `library` just yet.

Reply:
Well, I believe both are equally important :) Currently after generating the project, I always need to manually add libraries to it.